### PR TITLE
CLC-5472, New feature flag: webcast_sign_up_on_calcentral

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-5472

Modified logic in can_view_webcast_sign_up?. Feature flag will be false in upcoming release.

### CODE REVIEW

File: spec/policies/authentication_state_policy_spec.rb [hunk 1]
```diff
@@ -188,4 +186,36 @@
     end
   end
 
+  describe '#can_view_webcast_sign_up?' do
+    subject { AuthenticationState.new(session_state).policy.can_view_webcast_sign_up? }
+    context 'when feature flag is true' do
+      before { Settings.features.webcast_sign_up_on_calcentral = true }
```
Comment (raydavis) [inline spec/policies/authentication_state_policy_spec.rb]: @johncrossman , assigning the static variable adds unnecessary chanciness to the tests. I'd suggest configuring the flag as ``true`` in ``test.yml``, and using ``allow(Settings.features).to receive(:webcast_sign_up_on_calentral).and_return(false)`` to test that the flag works.

**Comment by raydavis** [comment]
RSpec failure.

File: spec/policies/authentication_state_policy_spec.rb (hunk 1 above)
Comment (johncrossman) [inline spec/policies/authentication_state_policy_spec.rb]: Thanks @raydavis  Fixed. I'll follow up on spec failure

**Comment by johncrossman** [comment]
Mr. T is happy

**Pull request merged**